3rdparty/mbedtls/README.md

```diff
@@ -16,13 +15,8 @@ changes:
   entropy implementation mbedTLS libraries to avoid a circular dependency
   with the Open Enclave core runtime.
 
-- It backports mbedtls patches from the development branch to the LTS version
-  OE currently uses:
+- It cherry-picks mbedtls patches from the development branch to the LTS
+  branch:
 
-  - Update `mbedtls/library/x509write_crt.c` with fixes from [#2632](
-    https://github.com/ARMmbed/mbedtls/pull/2632) contained in
-    `0001-Backport-2632-code-changes-to-2.16.patch`.
-
-  - Update `mbedtls/library/x509write_csr.c` with fixes from [#3464](
-    https://github.com/ARMmbed/mbedtls/pull/3464) contained in
-    `0001-Avoid-stack-allocation-of-large-memory-buffers.patch`.
+  - de7e036: Merge pull request #3489 from CodeMonkeyLeet/mbedtls-2.16_backport_3464
```

Comment:
Are we still cherry-picking these patches?

Reply:
Yes, the backports to mbedtls-2.16 branch were not merged until a few days ago, so did not make it into their 2.16.7 tag.